### ISSUE TITLE
LSP throws error when using `value` with URLSearchParams `delete()`

### ISSUE THREAD

**Issue by timharek** [denoland/deno#20295]
Following the [official docs for URLSearchParams](https://deno.land/api@v1.36.3?s=URLSearchParams#method_delete_1):
```typescript
let searchParams = new URLSearchParams([['name', 'value']]);
searchParams.delete('name');
searchParams.delete('name', 'value');
```

This:
```typescript
searchParams.delete('name', 'value'); 
```
gives the following error `Expected 1 arguments, but got 2.deno-ts(2554)`

**Comment by bartlomieju** [comment]
It appears the type declarations are outdated. I believe this will be fixed by an upgrade to TypeScript 5.2.

**Comment by lucacasonato** [comment]
This is now fixed
